dxvk.conf

```diff
@@ -339,6 +339,24 @@
 # environment variable is not set, otherwise it will be
 # ignored. The syntax is identical.
 
+# a valid value is : devinfo,fps,frametimes,pipelines,memory,version,api,compiler,
```

Comment:
If the above feedback is addressed then this is probably redundant and can be left out.

Reply:
A sample to guide out low level like me